cmd/crates/stellar-rpc-client/src/lib.rs

```diff
@@ -279,6 +279,24 @@ pub struct SimulateHostFunctionResult {
     pub xdr: xdr::ScVal,
 }
 
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone, PartialEq)]
+pub enum LedgerEntryChangeType {
+    #[serde(rename = "created")]
+    Created,
+    #[serde(rename = "deleted")]
+    Deleted,
+    #[serde(rename = "updated")]
+    Updated,
+}
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone)]
+pub struct LedgerEntryChange {
+    #[serde(rename = "type")]
+    pub type_: LedgerEntryChangeType,
+    pub key: String,
+    pub before: Option<String>,
+    pub after: Option<String>,
+}
```

Comment:
One more thought if you go this route:

It can be more ergonomic for the user of the enum to make a new struct for each variant's fields instead of having the fields directly inside each variant. For the user of the type it's easier to pass around the variant's data if it's already packaged in its own type. Otherwise the user needs to build their own container or pass around the fields separately. 

It's a shame that rust doesn't make enum variants their own types. If it did we could just get this for free.